src/generate-client.ts

```diff
@@ -147,7 +147,7 @@ function makeMethodParams(params: Params) {
 
 function makeReturnType(methodName: string, returnType) {
   // we have a json object, so we need to convert it to interface
-  if (returnType.toString() === '[object Object]') {
+  if (returnType && returnType.toString() === '[object Object]') {
```

Comment:
i think if the return type is `null` we should treat it as void !?
i will add that to the todo